lib/inferno/utils/ig_downloader.rb

```diff
@@ -0,0 +1,60 @@
+module Inferno
+  module Utils
+    module IgDownloader
+      FHIR_PACKAGE_NAME = /^[a-z][a-zA-Z0-9-]*\.([a-z][a-zA-Z0-9-]*\.?)*/
```

Comment:
I think these names are confusing and should end in `REG_EX` to make it clear that `FHIR_PACKAGE_NAME`, for example, isn't an actual package name.

Reply:
Fixed